aten/src/ATen/native/mkldnn/MKLDNNConversions.cpp

```diff
@@ -198,4 +199,49 @@ Tensor mkldnn_reorder_conv3d_weight(
 
 #endif // AT_MKLDNN_ENABLED()
 
+#if AT_MKL_ENABLED()
+#include <mkl.h>
+
+Tensor reorder_linear_weight(
+    const Tensor& weight,
+    const c10::optional<int64_t> batch_size) {
+  // TODO: bf16 prepack using MKLDNN
+  TORCH_CHECK(
+      weight.scalar_type() == ScalarType::Float,
+      "reorder_linear_weight: weight's dtype should be float");
+  c10::impl::ExcludeDispatchKeyGuard edkg(c10::autograd_dispatch_keyset);
+  auto M = batch_size.has_value() ? batch_size.value() : 128;
```

Comment:
Would setting `128` arbitrary work when the packed weight is used for MKL `cblas_sgemm_compute` with a different batch size?